mlflow/cli/eval_utils.py

```diff
@@ -0,0 +1,188 @@
+"""
+Utility functions for trace evaluation output formatting.
+"""
+
+from typing import Any
+
+import click
+import pandas as pd
+
+from mlflow.exceptions import MlflowException
+from mlflow.genai.scorers import get_all_scorers, get_scorer
+
+
+def resolve_scorers(scorer_names: list[str], experiment_id: str) -> list[Any]:
+    """
+    Resolve scorer names to scorer objects.
+
+    Checks built-in scorers first, then registered scorers.
+
+    Args:
+        scorer_names: List of scorer names to resolve
+        experiment_id: Experiment ID for looking up registered scorers
+
+    Returns:
+        List of resolved scorer objects
+
+    Raises:
+        click.UsageError: If a scorer is not found or no valid scorers specified
+    """
+    resolved_scorers = []
+    builtin_scorers = get_all_scorers()
+    builtin_scorer_map = {scorer.__class__.__name__: scorer for scorer in builtin_scorers}
```

Comment:
Scorers have a `name` field, and the built-in scorers use snake case values for `name`, e.g. `expectations_guidelines`, rather than CamelCase class names.

Are we expecting entries in `scorer_names` to be user-specified values? If so, we should use `scorer.name` instead of `scorer.__class__.name`.